index.php

```diff
@@ -126,7 +126,8 @@
 
     // redirect to ticket
     if (isset($_GET["redirect"])) {
-        Toolbox::manageRedirect($_GET["redirect"]);
+        global $_UGET;
+        Toolbox::manageRedirect($_UGET["redirect"]);
```

Comment:
```suggestion
        Toolbox::manageRedirect(Sanitizer::unsanitize($_UGET["redirect"]));
```

Reply:
Is this change meant to unsanitize the GET property instead of the UGET one?

Reply:
Indeed, you should unsanitize `$_GET`.
`$_U*` was introduced before sanitizer could perform a reverse non-destructive sanitization. These should not be used anymore.